pkg/mei/mei.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2012-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mei
```

Comment:
Write a package comment. I'd mention the device name /dev/mei0, and what kernel configs to turn on. We haven't been doing that and it'd be nice to start...

Reply:
done! I think all these files should also have a `_linux.go` extension since this is linux-specific